language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageLoader.java

```diff
@@ -116,8 +117,10 @@ public List<PackageInfo> getDistributionRepoPackages() {
      * @return {@link List} packages
      */
     public List<PackageInfo> getAllVisiblePackages(DocumentServiceContext ctx) {
-        List<PackageInfo> packagesList = new ArrayList<>();
-        packagesList.addAll(this.getDistributionRepoPackages());
+        Map<String, PackageInfo> packagesList = new HashMap<>();
+        this.getDistributionRepoPackages().forEach(packageInfo ->
+                packagesList.put(packageInfo.packageOrg.toString() + "/"
```

Comment:
We can use `packageInfo.getPackageIdentifier()` here right?

Reply:
Done